hw/ip/prim/dv/prim_alert/data/prim_alert_testplan.hjson

```diff
@@ -91,5 +91,20 @@
               "prim_sync_fatal_alert"]
     }
 
+    {
+      name: prim_alert_gate_sender_clk_rst_test
+      desc: '''Gate prim_alert_sender clk or reset during alert_handshake.
+
+            - During alert handshake, gate prim_alert_sender's clock or issue reset.
+            - Verify alert handshake can resume after prim_alert_sender's re-initialized.
+            - If clock was gated, verify alert handshake will resume when clock is active,
```

Comment:
Could you be more explicit about checking we will never miss an alert when the clock is gated.

Reply:
sure I will add that, thanks Matute.